server/cluster/cluster.go

```diff
@@ -267,33 +275,41 @@ func (c *Cluster) updateShardVersionLocked(shardID uint32, prevVersion, newVersi
 	}, nil
 }
 
-func (c *Cluster) getShardTopologyFromStorage(ctx context.Context, shardID uint32) (*clusterpb.ShardTopology, error) {
-	shardTopologies, err := c.storage.ListShardTopologies(ctx, c.clusterID, []uint32{shardID})
+func (c *Cluster) getShardViewFromStorage(ctx context.Context, shardID uint32) (*clusterpb.ShardTopology, error) {
```

Comment:
Same as above, there is two struct `ShardTopology` and `ShardView`, and they are often confused.
```suggestion
func (c *Cluster) getShardTopologyFromStorage(ctx context.Context, shardID uint32) (*clusterpb.ShardTopology, error) {
```